extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -126,28 +157,23 @@ def upload_file() -> typing.Tuple[str, int]:
         flask.abort(HTTPStatus.BAD_REQUEST)
 
     try:
-        file_z = zipfile.ZipFile(file)
+        zipfile.ZipFile(file)
+        file.seek(0)
```

Comment:
Are you seeking here because `zipfile.ZipFile` advances the file handle?

Reply:
yes